Model/Cron.php

```diff
@@ -239,6 +249,36 @@ class Cron
      */
     protected $configHelper;
 
+    /**
+     * @var
+     */
+    protected $_recurringDetailReference;
+
+    /**
+     * @var
+     */
+    protected $_expiryDate;
+
```

Comment:
same here
```suggestion
```

Reply:
Indeed it can be removed in both locations